middlewares/authentication.js

```diff
@@ -49,8 +116,8 @@ const authMiddleware = {
   name: 'ensureEvanAuth',
   global: false,
   priority: 10,
-  preProcessor: ({ connection }) => {
-    _ensureAuth(connection)
+  preProcessor: async ({ connection }) => {
+    await _ensureAuth(connection)
```

Comment:
❓ doesn't this also work?
```suggestion
  preProcessor: _ensureAuth,
```